app/controllers/api/v1/medias_controller.rb

```diff
@@ -270,10 +258,10 @@ def destroy
         render json: { status: 'SUCCESS', message: 'Deleted successfully', data: @media }, status: :ok
       end
 
-		private
-		def media_params
-      params.permit(:media_type)
-    end
-  end
-end
+			private
```

Comment:
This indentation is showing correct in my system. I am not getting why it is different here